.github/labeler.yml

```
@@ -0,0 +1,30 @@
+- name: Labeler
+  uses: actions/labeler@v3
+lang/ko:
+- content/ko/*
+lang/bn:
+- content/bn/*
```

I suggest to put empty lines between languages.

Plus, it would be better to list language entries in ascending alphabetical order.
It will make easy to
- find existing language entry
- add new language entry

```suggestion
lang/ko:
- content/ko/*

lang/bn:
- content/bn/*
```